__tests__/app.test.js

```diff
@@ -44,6 +44,35 @@ describe("/api/topics", () => {
   });
 });
 
+describe("/api/articles", () => {
+  test("GET:200 sends an array of article objects, sorted by date in descending order by default", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toEqual(expect.any(Array));
+        expect(articles.length).toBe(13);
+        articles.forEach((article) => {
+          expect(Object.keys(article).length).toEqual(8);
+          expect(Object.keys(article)).toEqual(
+            expect.arrayContaining([
```

Comment:
This is fine, but it can be helpful to also check what values are expected in terms of data type e.g. will the id be an integer? will the author be a string? etc